{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -17,6 +17,8 @@ exclude = '''
 )
 '''
 
+{% if cookiecutter.linter_name == "pylint" %}
 [tool.isort]
```

Comment:
I think *isort* is running in a separate pre-commit check and is not related to pylint.